trillian/util/log_leaf.go

```diff
@@ -71,3 +50,57 @@ func ExtraDataForChain(cert ct.ASN1Cert, chain []ct.ASN1Cert, isPrecert bool) ([
 	}
 	return tls.Marshal(extra)
 }
+
+func BuildLogLeafWithHash(logPrefix string, merkleLeaf ct.MerkleTreeLeaf, leafIndex int64, cert ct.ASN1Cert, chain []ct.ASN1Cert, hash []byte, isPrecert bool) (trillian.LogLeaf, error) {
```

Comment:
nit: `hash` -> `chainHash` to be more obvious what it is?

Reply:
Done.